### PR TITLE
Cleanup old hbase compact modules

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -298,17 +287,6 @@
           </exclusion>
         </exclusions>
       </dependency>
-      <dependency>
-        <groupId>org.apache.tephra</groupId>
-        <artifactId>tephra-hbase-compat-1.0</artifactId>
```
Comment (yaojiefeng) [inline pom.xml:303]: Why do we remove this? I think only 0.96 and 0.98 compat are removed right?

**Review by yaojiefeng — APPROVED** [review]
Just one question, rest LGTM

**Comment by chtyim** [comment]
RAT passed https://builds.cask.co/browse/CDAP-DRC9476

File: pom.xml (hunk 1 above)
Comment (chtyim) [reply pom.xml:303]: It was a duplicate as reported by IDE.

**Pull request merged**